import-export-cli/integration/README.md

```diff
@@ -0,0 +1,97 @@
+# APICTL Integration testing
+
+## Pre-requisites for running integration tests
+1. In order to run the integration tests, two instances of API Manager need to be started. These two instances will act as the APIM environments that will be used for doing exporting and importing scenarios respectively. The integration tests will execute apictl commands, DCR calls and REST API calls against these instances.
+
+2. Before starting the APIM instances update the APIM *deployment.toml* to allow the `@` character as part of the service provider name. This is to support DCR for tenant users since the `@` is part of the username. The updated ocnfig should look as follows,
```

Comment:
spelling mistake in config

Reply:
Fixed in https://github.com/wso2/product-apim-tooling/pull/310/commits/18287b957368281440c9d4da82ebaf51636407e2